firmware/lib/globalTime/globalTime.cpp

```diff
@@ -40,9 +40,13 @@ void GlobalTime::updateTime() {
 
         m_day = day(m_unixEpoch);
         m_month = month(m_unixEpoch);
-        m_monthName = monthStr(m_month);
+//PSC 10/22/2024 Added Monthname in German
+//        m_monthName = monthStr(m_month);
+        m_monthName = LOC_MONTH[m_month-1];
```

Comment:
Please remove both the comment about what you are changing and the old version. That is what version control is for.